Cargo.toml

```diff
@@ -17,8 +17,13 @@ categories = ["network-programming"]
 keywords = ["mdns", "dns", "multicast", "chromecast", "discovery"]
 
 [dependencies]
-mio = "0.6"
 dns-parser = "0.8"
 net2 = "0.2"
 error-chain = "0.12"
 get_if_addrs = "0.5.3"
+futures = "0.1.28"
+tokio-udp = "0.1.3"
```

Comment:
Recommend locking to minor revision only like `futures = "0.1"` so we are compatible with `0.1.*` and don't force crate users to include two versions of a crate.